libr/core/cmd_debug.c

```diff
@@ -1331,85 +1331,37 @@ static void cmd_debug_modules(RCore *core, int mode) { // "dmm"
 		case 0:
 			r_cons_printf ("0x%08"PFMT64x" %s\n", map->addr, map->file);
 			break;
-		case ':':
-			if (addr >= map->addr && addr < map->addr_end) {
-#if __WINDOWS__ && !__CYGWIN__
-				/* Escape backslashes in the file path on Windows */
-				char *escaped_path = r_str_escape (map->file);
-				char *escaped_name = r_str_escape (map->name);
-				if (escaped_path && escaped_name) {
-					r_name_filter (escaped_name, 0);
-					r_cons_printf ("f mod.%s = 0x%08"PFMT64x"\n",
-							escaped_name, map->addr);
-					r_cons_printf (".!rabin2 -rsB 0x%08"PFMT64x" \'%s\'\n",
-							map->addr, escaped_path);
-				}
-				free (escaped_path);
-				free (escaped_name);
-#else
-				char *fn = strdup (map->file);
-				r_name_filter (fn, 0);
-				//r_cons_printf ("fs+module_%s\n", fn);
-				r_cons_printf ("f mod.%s = 0x%08"PFMT64x"\n",
-						fn, map->addr);
-				r_cons_printf (".!rabin2 -rsB 0x%08"PFMT64x" '%s'\n",
-						map->addr, map->file);
-				//r_cons_printf ("fs-\n");
-				free (fn);
-#endif
-			}
-			break;
 		case '.':
 			if (addr >= map->addr && addr < map->addr_end) {
 				r_cons_printf ("0x%08"PFMT64x" %s\n", map->addr, map->file);
 				goto beach;
 			}
 			break;
 		case 'j':
-#if __WINDOWS__ && !__CYGWIN__
 			{
-				/* Single backslashes cause issues when parsing JSON output, so escape them */
+				/* Escape backslashes (e.g. for Windows). */
```

Comment:
Why did you remove `__WINDOWS__ && !__CYGWIN__`? Escaping backslashes is not required for non-Windows system.

Reply:
It shouldnt hurt

Reply:
Why? It hurts.